osdep/android/posix-spawn.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * posix-spawn replacement for Android
+ *
+ * This file is part of mpv.
+ *
+ * mpv is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * mpv is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef MP_OSDEP_ANDROID_POSIX_SPAWN
+#define MP_OSDEP_ANDROID_POSIX_SPAWN
+
+#include <sys/types.h>
+
+#define MAX_FILE_ACTIONS 4
+
+typedef struct {} posix_spawnattr_t; /* unsupported */
```

Comment:
Compilers might complain about empty struct, so add `char dummy;` or something to it. Or remove `{}` to make it an opaque struct.